addons/udes_stock/models/stock_picking_type.py

```diff
@@ -334,6 +334,14 @@ class StockPickingType(models.Model):
         " for any empty picking in the system.",
     )
 
+    # Maybe `reserve_pallet_per_picking` as we don;t know about sale orders in
```

Comment:
typo: don;t

Reply:
I've removed the comment, as it was redundant anyway